controllers/articleController.js

```diff
@@ -0,0 +1,153 @@
+import dotenv from 'dotenv';
+import models from '../models';
+
+const { articles, users } = models;
+const clientUrl = (process.env.NODE_ENV === 'production') ? 'https://ah-92explorers-api.herokuapp.com' : 'http://127.0.0.1:3000';
+
+dotenv.config();
+
+/**
+ * @param {class} --Article controller
+ */
+class ArticleController {
+  /**
+   *
+   * @param {Object} req
+   * @param {Object} res  get all created articles
+   * @returns {Object} return all created articles
+   */
+  static async index(req, res) {
+    try {
+      // @retrieve articles
+      const allArticles = await articles.findAndCountAll({
+        order: [['article_id', 'DESC']],
+        include: [
+          {
+            model: users,
+            as: 'authorfkey',
```

Comment:
this will return `authorfkey` instead of `author`. kindly change that so that it reflects the meaning of the return key